IndyAV/simulation/indyav_gazebo/xacro/wheels.xacro

```diff
@@ -2,93 +2,68 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro" name="wheels">
   <xacro:include filename="$(find mil_gazebo)/xacro/util.xacro"/>
 
-  <xacro:property name="wheel_length" value="0.25"/>
-  <xacro:property name="wheel_width" value="0.25"/>
-  <xacro:property name="wheel_height" value="0.25"/>
+  <xacro:property name="wheel_radius" value="0.125"/>
+  <xacro:property name="wheel_width" value="0.125"/>
   <xacro:property name="wheel_mass" value="5"/>
 
-  <xacro:macro name="wheel_geometry">
+  <xacro:macro name="wheel_geometry">	  
     <geometry>
-      <box size="${wheel_length} ${wheel_width} ${wheel_height}"/>
+       <cylinder length="${wheel_width}" radius="${wheel_radius}"/>
     </geometry>
   </xacro:macro>
 
-  <xacro:macro name="wheel_surface">
-    <surface>
-      <friction>
-        <ode>
-          <mu>1000</mu> <!-- left / right  friction ,very high -->
-          <mu2>1</mu2> <!-- forwards / backwards friction, very low -->
-          <fdir1> 0 1 0 </fdir1>
-        </ode>
-      </friction>
-    </surface>
-  </xacro:macro>
-
   <xacro:macro name="do_wheel_surface" params="name">
     <gazebo reference="${name}_link">
-      <collision>
-        <xacro:wheel_surface/>
-      </collision>
+      <mu1>500</mu1> <!-- friction parallel to the tire (wheels need friction to roll) -->
+      <fdir1> 0 1 0 </fdir1> <!--States the direction of mu1-->
     </gazebo>
   </xacro:macro>
 
   <xacro:macro name="wheel_physical">
     <inertial>
+      <origin rpy="0 1.5708 0"/>
       <mass value="${wheel_mass}"/>
```

Comment:
you only need to list `<origin rpy="0 1.5708 0"/>` once if you do it higher up one level and it will apply it to all lower level features